.travis.yml

```diff
@@ -3,5 +3,11 @@ cache: bundler
 rvm:
   - 2.3.1
 
-install: bundle install
-script: jekyll build --safe
+before_script: gem install awesome_bot
+script: awesome_bot _posts/*.md --allow-dupe --allow-redirect
```

Comment:
Can't we put all the files in 1 line? Something like

script: awesome_bot _posts/*.md, support/*.md, contact/*.md, pulse/*.md  --allow-dupe --allow-redirect